mango/data/__init__.py

```diff
@@ -3,6 +3,7 @@
 import pandas
 
 from .calendar_data import get_calendar
+from .series_correlations import get_corr_matrix
```

Comment:
I think this kind of functions would be better in the processing module, in the processing_time _series.py script.
@AntonioGonzalezSuarez @ggsdc What do you think?